nautobot/ipam/tests/test_models.py

```diff
@@ -159,7 +160,7 @@ def test_get_db_prep_value_mysql(self):
         self.assertEqual(prepped, manual)
 
 
-class TestPrefix(TestCase):
+class TestPrefix(TestCase):  # TODO change to BaseModelTestCase
```

Comment:
Sorry but whats the reason here for not changing it to `BaseModelTestCase`? I think we do have a `PrefixFactory`.

Reply:
See line 165/166 below -- this test case currently does a `Prefix.objects.all().delete()` as part of its `setUp()` and then only creates individual Prefix instances in specific test steps -- so the generic test wouldn't be able to use the factory Prefixes, unfortunately.